### PR TITLE
Sms 26 set background color

### PR DESCRIPTION
**Pull request by JohnNStout**
This adds the prompt painter view and lets the user select a background color before drawing.

### CODE REVIEW

File: mytestvueapp.client/src/router/index.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ const router = createRouter({
       name: "Painter View",
       component: () => import("../views/PainterView.vue"),
     },
+    {
+      path: "/prompt",
```
Comment (GranquistR) [inline mytestvueapp.client/src/router/index.ts:22]: can we make this more descriptive? maybe call the path something like /create or /new?

File: mytestvueapp.client/src/utils/codec.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default class codec {
     height: number,
     width: number
   ): PixelGrid {
-    const decodedPicture: PixelGrid = new PixelGrid(height, width);
+    const decodedPicture: PixelGrid = new PixelGrid(height, width,);
```
Comment (GranquistR) [inline mytestvueapp.client/src/utils/codec.ts:20]: Syntax Error?

File: mytestvueapp.client/src/views/PainterView.vue [hunk 3]
```diff
@@ -36,22 +36,30 @@ import BrushSelection from "@/components/PainterUi/BrushSelection.vue";
 import ColorSelection from "@/components/PainterUi/ColorSelection.vue";
 import PainterTool from "@/entities/PainterTool";
 import { Vector2 } from "@/entities/Vector2";
+import PromptPainter from "@/views/PromptPainter.vue"
+import { getConstantValue } from "typescript";
```
Comment (GranquistR) [inline mytestvueapp.client/src/views/PainterView.vue:40]: Is this import being used? If not, use it.

File: mytestvueapp.client/src/views/PromptPainter.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+localStorage.setItem('backgroundColor', '#ffffff');
+
+const backgroundColor = defineModel<string>({ default: "#ffffff" });
+function updateLocalStorage(){
+  localStorage.setItem('backgroundColor', backgroundColor.value);
+}
+</script>
+<style>
+.p-colorpicker-preview {
+  width: 7rem !important;
```
Comment (GranquistR) [inline mytestvueapp.client/src/views/PromptPainter.vue:34]: Convert these styles to [PrimeFlex!](https://primeflex.org/)

**Review by GranquistR — CHANGES_REQUESTED** [review]

File: mytestvueapp.client/src/router/index.ts (hunk 1 above)
Comment (JohnNStout) [reply mytestvueapp.client/src/router/index.ts:22]: changed to /new

File: mytestvueapp.client/src/views/PainterView.vue (hunk 3 above)
Comment (JohnNStout) [reply mytestvueapp.client/src/views/PainterView.vue:40]: Don't need so deleted.

File: mytestvueapp.client/src/views/PromptPainter.vue (hunk 4 above)
Comment (JohnNStout) [reply mytestvueapp.client/src/views/PromptPainter.vue:34]: Fixed. please look over.

**Review by JacobSletten — APPROVED** [review]
Looks good to me

**Review by JacobSletten — APPROVED** [review]
Still looks good to me.

File: mytestvueapp.client/src/utils/codec.ts (hunk 2 above)
Comment (JohnNStout) [reply mytestvueapp.client/src/utils/codec.ts]: fixed

**Pull request merged**